spark/src/main/scala/geotrellis/spark/ingest/Pyramid.scala

```diff
@@ -15,15 +13,15 @@ object Pyramid extends Logging {
   /**
    * Functions that require RasterRDD to have a TMS grid dimension to their key
    */
-  def up[K: SpatialComponent: ClassTag](rdd: RasterRDD[K], level: LayoutLevel, layoutScheme: LayoutScheme): (LayoutLevel, RasterRDD[K]) = {
+  def up[K: SpatialComponent: ClassTag](rdd: RasterRDD[K], layoutScheme: LayoutScheme, zoom: Int): (Int, RasterRDD[K]) = {
     val metaData = rdd.metaData
-    val nextLevel = layoutScheme.zoomOut(level)
+    val LayoutLevel(nextZoom, nextLevel) = layoutScheme.zoomOut(LayoutLevel(zoom, rdd.metaData.layout))
     val nextMetaData = 
       RasterMetaData(
         metaData.cellType,
-        metaData.extent,
-        metaData.crs,
-        nextLevel.tileLayout
+        nextLevel,
+        metaData.dataExtent, // TODO no way, these have to change
```

Comment:
Stale, will remove. Used to hold GridBounds, which would have had to be recollected at this point. Holding the extent avoids the need for recalculation on zoom out/in.